src/main/java/org/symphonyoss/symphony/messageml/elements/Button.java

```diff
@@ -105,6 +105,12 @@ public void validate() throws InvalidInputException {
       throw new InvalidInputException("Attribute \"class\" must be \"primary\", \"secondary\", " +
               "\"tertiary\" or \"destructive\" (\"primary-destructive\" and \"secondary-destructive\" are deprecated)");
     }
+    if("primary-destructive".equals(clazz)) {
```

Comment:
Hi @lmacedodaitan why in the validate() ? It is not better to put the logic in buildAttributes? BTW doing in this way I suppose that DUPLICATES attribute, instead of replacing...

Reply:
Yes, it seems right to put inside the buildAttributes method, thanks. But it won't duplicate as it is a map of attributes, the values will be replaced with newer ones.